daemon/logger/copier.go

```diff
@@ -94,9 +101,23 @@ func (c *Copier) copySrc(name string, src io.Reader) {
 				default:
 					msg := NewMessage()
 					msg.Source = name
-					msg.Timestamp = time.Now().UTC()
 					msg.Line = append(msg.Line, buf[p:p+q]...)
 
+					if hasMorePartial {
+						msg.PLogMetaData = &types.PartialLogMetaData{ID: partialid, Ordinal: ordinal, Last: true}
```

Comment:
Can we keep a `PartialLogMetaData` around to re-use... or usage a sync/pool?
I think just keeping a single one should work.

Reply:
Since the copy is sequential, I'm now simply using one allocated object over all msgs. Would that work for you?

Reply:
Done!